frontend/src/components/Button/Button.test.jsx

```diff
@@ -0,0 +1,27 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Button } from 'components/Button/Button';
+
+describe('Button', () => {
+  it('properly render component', () => {
+    //when
+    render(<Button buttonName={'SampleButtonName'} />);
+    const receivedComponentText = screen.getByText('SampleButtonName');
+    const receivedComponentByRole = screen.getByRole('button');
+
+    //then
+    expect(receivedComponentText).toBeInTheDocument();
+    expect(receivedComponentByRole).toBeInTheDocument();
+  });
```

Comment:
Tutaj taka uwaga na przyszłość. Sam `getBy...` już sprawdza, bo jak nie znajdzie, to wyrzuci błąd i ustawi test, że Fail - więc też jakby sprawdza. Więc raczej "then".
Tak na przyszłość to ;)


```suggestion
  it('properly render component', () => {
    //when
    render(<Button buttonName={'SampleButtonName'} />);

    //then
    const receivedComponentText = screen.getByText('SampleButtonName');
    const receivedComponentByRole = screen.getByRole('button');
    expect(receivedComponentText).toBeInTheDocument();
    expect(receivedComponentByRole).toBeInTheDocument();
  });
```

A na przyszłość to `expect(receivedComponentByRole).toBeInTheDocument();` możesz właściwie wyrzucić, bo skoro zostało znalezione, to znaczy, że istnieje.

Reply:
To w takim przypadku nie używać expecta?

Reply:
Wiesz co, może, żebym Wam nie namieszał.
Myślę, że używajcie!

Nie jest błędem użycie, więc raczej użyjcie. Trochę się rozpędziłem, gdyż u mnie w pracy zwykle się nie używa, ale to może być u nas w firmie taki zwyczaj. A nie wiem czy nieużycie jest zawsze ok.

Przepraszam, że mogłem wprowadzić w błąd. 
Z `.toBeInTheDocument();` na pewno jest dobrze!